src/vanna/base/base.py

```diff
@@ -67,6 +67,8 @@
 from ..types import TrainingPlan, TrainingPlanItem
 from ..utils import validate_config_path
 
+MAX_TOKENS = 14000
```

Comment:
@zainhoda Instead of having this variable here, wouldn't it make more sense to have it as a private variable inside the `VannaBase` class, if the goal is to use it there?